templates/openshift/common/route/main.yml.j2

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     env_type:         "{{ env_type }}"
     customer:         "{{ customer }}"
-    app:              "{{ app.name }}-{{ deployment_stamp | default('init', true) }}"
+    app:              "{{ app.name }}"
     version:          "{{ item.0.tag | default('init', true) }}"
     deployment_stamp: "{{ deployment_stamp | default('init', true) }}"
```

Comment:
The `init` deployment stamp does not exist anymore.

Reply:
Yes I now, I can change for `null` if you want.